pnpm-lock.yaml

```diff
@@ -1,372 +1,272 @@
-lockfileVersion: '6.0'
+lockfileVersion: 5.4
```

Comment:
Looks like you used a newer version of `pnpm` here 📜   We should all update so our `pnpm-lock` files will look the same when generating.

Reply:
Ah interesting, I didn't do anything to upgrade, must have been some auto-update. I'll revert.